src/test/ui/const-generics/min_const_generics/macro.rs

```diff
@@ -0,0 +1,47 @@
+// run-pass
+#![feature(min_const_generics)]
+
+struct Example<const N: usize>;
+
+macro_rules! external_macro {
+  () => {{
+    const X: usize = 1337;
+    X
+  }}
+}
+
+trait Marker<const N: usize> {}
+impl<const N: usize> Marker<N> for Example<N> {}
+
+fn make_marker() -> impl Marker<{ gimme_a_const!(marker) }> {
```

Comment:
can you add and test a new `#[macro_export]` here too?

We lower argument and return impl trait differently and previously had some bugs there so don't want to miss anything here.

After this r=me